build.gradle

```diff
@@ -68,10 +65,12 @@ project(":core") {
         api "com.badlogicgames.gdx:gdx-freetype:$gdxVersion"
         api "com.esotericsoftware.spine:spine-libgdx:4.0.18.1"
         api "com.github.MrStahlfelge.gdx-websockets:core:1.9.10.3"
-        testImplementation 'org.junit.jupiter:junit-jupiter-api:5.3.1'
-        testImplementation("org.assertj:assertj-core:3.11.1")
-        implementation("org.assertj:assertj-core:3.11.1")
```

Comment:
usunąłem to, bo w sumie nie wiedziałem co to jest, ale faktycznie może być fajne